### PR TITLE
Update to Character Class

### PR DESCRIPTION
**Pull request by maliceholmes**
Addition of Character Json. And I think this works now

### CODE REVIEW

File: Character.py [hunk 1]
```diff
@@ -1,18 +1,54 @@
...
+        #self.Name = random.choice(['Kevin', 'Alice', 'Dri', 'J'])
         pass
 
+
+    def isProficient(self, skill):
+        for k in self.Proficiencies:
+            if k == skill:
+                return True
+        return 
+
```
Comment (kpwelsh) [inline Character.py:13]: This compares strings. Not Enum values.

File: Character.py [hunk 2]
```diff
@@ -1,18 +1,54 @@
 import random
+import json
 
 class Character:
     def __init__(self):
-        self.Name = random.choice(['Kevin', 'Alice', 'Dri', 'J'])
+        #self.Name = random.choice(['Kevin', 'Alice', 'Dri', 'J'])
         pass
 
+
```
Comment (kpwelsh) [inline Character.py:10]: Same thing as below

File: Character.py [hunk 3]
```diff
@@ -1,18 +1,54 @@
...
+    def isProficient(self, skill):
+        for k in self.Proficiencies:
```
Comment (kpwelsh) [inline Character.py]: Instead of looping through all of these things, you can just use the keyword 'in'

File: Character.py [hunk 4]
```diff
@@ -1,18 +1,54 @@
...
+        for k in self.Expertise:
+            if k == skill:
+                return True
+        return
+
+    
+
     @classmethod
-    def fromJson(cls, jsonObj):
+    def fromJson(cls):
```
Comment (kpwelsh) [inline Character.py]: This should take some kind of parameter to specify which character to load.

**Review by kpwelsh — CHANGES_REQUESTED** [review]

File: Character.py (hunk 3 above)
Comment (kpwelsh) [reply Character.py]: ex
`return (skill in self.Proficiencies)`

File: Character.py (hunk 4 above)
Comment (maliceholmes) [reply Character.py]: >             This should take some kind of parameter to specify which character to load.

Like (self, skill, name)?

**Pull request closed without merging**